keyboards/keychron/k4/rgb/v2/ansi/led.c

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 PythonDeployer (@PythonDeployer)
```

Comment:
this file does not get included at build time. Why do you need it?

Reply:
Idk i guess added it with debugging and forgot to remove, ill delete cuz not needed :)